console/src/app/modules/policies/login-policy/login-policy.component.ts

```diff
@@ -285,6 +294,12 @@ export class LoginPolicyComponent implements OnInit, OnDestroy {
   }
 
   public savePolicy(): void {
+    if (this.lifetimeForm.invalid) {
+      // Display error message
+      this.toast.showError(Error('Form contains invalid value(s).'));
```

Comment:
Maybe the error should come from a static file. Let me know.

Reply:
yes I've commited the translations to the static files.. thx 👍